nntrainer/layers/lstm.h

```diff
@@ -125,17 +122,17 @@ class LSTMLayer : public Layer {
   unsigned int unit;
 
   /**
-   * @brief     activation function for h_t : default is tanh
+   * @brief     activation function for h_t : default is sigmoid
    */
   ActiFunc acti_func;
 
   /**
-   * @brief     activation type for recurrent : default is sigmoid
+   * @brief     activation type for recurrent : default is tanh
```

Comment:
Could you once checkout the terms?
It seems activation is tanh, recurrent_activation is sigmoid.
https://keras.io/api/layers/recurrent_layers/lstm/

Reply:
it is properly implemented in code. update the comment.

Reply:
Okay :)